Rakefile

```diff
@@ -63,18 +67,19 @@ namespace :test do
     sh("cd #{integration_dir}/build/ && terraform destroy -force")
   end
 
-  task :destroy_test_environment do
+  task :destroy_test_environment, :namespace do |t, args|
     puts "----> Destroying terraform environment"
     sh("cd #{integration_dir}/build/ && terraform env select default")
-    sh("cd #{integration_dir}/build && terraform env delete #{terraform_env}")
+    sh("cd #{integration_dir}/build && terraform env delete #{args[:namespace]}")
   end
 
   task :integration do
-    Rake::Task["test:configure_test_environment"].execute
+    namespace = ENV['INSPEC_TERRAFORM_ENV'] || prompt("Please enter a namespace for your integration tests to run in: ")
```

Comment:
This made it into our README from the original "create separate env" issue.  Done!